static/scss/partials/dashboard.scss

```diff
@@ -5,6 +5,8 @@
 
 .mpp-dashbaord-header-name {
     font-weight: 700;
+    width:100%;
```

Comment:
Is the `width` here necessary?

Reply:
Yes, it helps keep the word-wrap property valid on mobile.
https://user-images.githubusercontent.com/13066134/127021103-30bc7aff-5b4b-4c35-9917-b5d950b56b26.mov